YOLOX/README.md

```diff
@@ -0,0 +1,255 @@
+<div align="center"><img src="https://github.com/Megvii-BaseDetection/YOLOX/blob/main/assets/logo.png" width="350"></div>
+<img src="https://github.com/Megvii-BaseDetection/YOLOX/blob/main/assets/demo.png" >
+
+### Check out the repo on <a href="https://dagshub.com/arnavr.neo/YOLOX">DagsHub.</a>
+
+## Introduction
+YOLOX is an anchor-free version of YOLO, with a simpler design but better performance! It aims to bridge the gap between research and industrial communities.
+For more details, please refer to our [report on Arxiv](https://arxiv.org/abs/2107.08430).
+
+This repo is an implementation of PyTorch version YOLOX, there is also a [MegEngine implementation](https://github.com/MegEngine/YOLOX).
+
+<img src="https://github.com/Megvii-BaseDetection/YOLOX/blob/main/assets/git_fig.png" width="1000" >
+
+## Updates!!
+* 【2022/04/14】 We suport jit compile op.
+* 【2021/08/19】 We optimize the training process with **2x** faster training and **~1%** higher performance! See [notes](docs/updates_note.md) for more details.
+* 【2021/08/05】 We release [MegEngine version YOLOX](https://github.com/MegEngine/YOLOX).
+* 【2021/07/28】 We fix the fatal error of [memory leak](https://github.com/Megvii-BaseDetection/YOLOX/issues/103)
+* 【2021/07/26】 We now support [MegEngine](https://github.com/Megvii-BaseDetection/YOLOX/tree/main/demo/MegEngine) deployment.
+* 【2021/07/20】 We have released our technical report on [Arxiv](https://arxiv.org/abs/2107.08430).
+
+## Coming soon
+- [ ] YOLOX-P6 and larger model.
+- [ ] Objects365 pretrain.
+- [ ] Transformer modules.
+- [ ] More features in need.
+
+## Benchmark
+
+#### Standard Models.
```

Comment:
All links below should be modified to point to the corresponding model files you uploaded to DagsHub

Reply:
The links have been updated.